glean-core/ffi/src/lib.rs

```diff
@@ -52,6 +53,21 @@ pub extern "C" fn glean_initialize(data_dir: FfiStr, application_id: FfiStr) ->
             log::debug!("Android logging should be hooked up!")
         });
     }
+    // Make sure logging does something on non Android platforms as well. Use
+    // the RUST_LOG environment variable to set the desired log level, e.g.
+    // setting RUST_LOG=debug sets the log level to debug.
+    #[cfg(not(target_os = "android"))]
+    {
+        match env_logger::try_init() {
+            Ok(_) => log::debug!("stdout logging should be hooked up!"),
+            Err(_) => log::debug!("stdout was already initialized"),
```

Comment:
It's ... not very helpful to log the error that the log setup failed using the same log mechanism.
IMO it's fine to call `env_logger::init()` here, it will panic if that fails, therefore ultimately killing the process.

Reply:
This is only supposed to fail in tests, where it will try to init the logger multiple times. If that panics, the test itself will fail. 

When this fails, the logger should still be up and running, given that was previously initialized already.

Reply:
Hm, fair enough. Guess that makes sense then. We should just keep it in mind once we add iOs to the mix, we will then need new `#[cfg...]`s anyway